### PR TITLE
PENDING UPDATE: Added line and crosshair documentation

### PR DESCRIPTION
**Pull request by ColinEberhardt**

### CODE REVIEW

File: tools/_posts/2010-10-10-crosshairs.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+                return crosshairData;
+        }
+    });
+
+  container.append('g')
+      .datum(data)
+      .call(multi);
+---
+
+The crosshair component renders a interactive crosshair. The component should be data-joined to an empty array which it will populate with an object that idicates the current crosshair location. This is a useful mechanism for displaying the crosshair location in some form of legend.
```
Comment (chrisprice) [inline tools/_posts/2010-10-10-crosshairs.md]: renders *an* interactive

Comment (chrisprice) [inline tools/_posts/2010-10-10-crosshairs.md]: i*n*dicates

File: tools/_posts/2010-10-10-line-annotation.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
+---
+layout: default
+title: Line Annottion
```
Comment (chrisprice) [inline tools/_posts/2010-10-10-line-annotation.md]: Annotation

File: tools/_posts/2010-10-10-crosshairs.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
+---
+layout: default
+title: Crosshairs
```
Comment (chrisprice) [inline tools/_posts/2010-10-10-crosshairs.md]: We should remember these when doing #387

**Comment by ColinEberhardt** [comment]
Thanks chris - just realised sublime isn't spell checking markdown files.

**Pull request merged**